src/main/java/fi/okm/jod/yksilo/config/suomifi/Saml2LoginConfig.java

```diff
@@ -95,6 +101,7 @@ SecurityFilterChain samlSecurityFilterChain(
                     request.logoutRequestResolver(logoutRequestResolver);
                   });
             })
+        .logout(logout -> logout.addLogoutHandler(profileDeletionHandler))
```

Comment:
I guess the mock login should also implement the deletion (would also allow testing the deletion).